### DISCUSSION TITLE
Can't export replays on mobile.

### DISCUSSION THREAD

**Discussion by TheKnuxlesGuy** [ppy/osu#22869] [Q&A] [answered]
So, I wanted to export one of my replays, but because I need to right click to do so and I'm on mobile, I can't really do that. I even tried to plug in a mouse into my phone but that didn't work, the right mouse button just worked as a back button (it's an Android thing afaik). Please, can anybody help me?
osu! version: 2023.305.0-lazer

**Comment by peppy** [comment, marked as answer]
We'll be adding the ability to right click by long-tapping in the future.

Until then you will likely just have to wait.

**Reply by TheKnuxlesGuy** [reply]
Ok, thanks anyway.
